lib/cli.rb

```diff
@@ -546,49 +535,71 @@ def self.check_port_validity(port)
       desc: "Print inspection result"
     c.switch :verbose, required: false, negatable: false,
       desc: "Display the filters which are used during inspection"
+  end
 
-    c.action do |global_options,options,args|
-      host = shift_arg(args, "HOSTNAME")
-      inspector_task = InspectTask.new
-      scope_list = process_scope_option(options[:scope], options["exclude-scope"])
-      name = options[:name] || host
+  def self.parse_inspect_command_options(host, options)
```

Comment:
Assignment Branch Condition size for parse_inspect_command_options is too high. [33.62/15]